examples/opt_serving/interactive_hosted.py

```diff
@@ -235,6 +313,63 @@ def completions(engine=None):
     return OAIResponse(results).__dict__()
 
 
+@app.route("/logprobs", methods=["POST"])
```

Comment:
Can we have a flag to turn on/off this API when starting the interactive host? And also, can we set a password or security key so that we can only allow private access to this API?

Reply:
Yeah I can definitely look into adding a flag and have it off by default or something. I'm not sure what's the best practice for the password/security key though, is there some method you have in mind?